tools/devtool

```diff
@@ -1442,6 +1458,74 @@ cmd_install() {
     install -m 755 "$CARGO_SECCOMPILER_TARGET_DIR/$target/$profile/seccompiler-bin" "$install_path"
 }
 
+# `./devtool build_rootfs -s 500M`
+# Build a kernel using the provided config file.
+#
+cmd_build_rootfs() {
+    # Default size for the resulting rootfs image is 1G.
+    SIZE="500M"
+
+    # Parse the command line args.
+    while [ $# -gt 0 ]; do
+        case "$1" in
+            "-h"|"--help")      { cmd_help; exit 1; } ;;
+            "-s"|"--size")
+                shift
+                SIZE="$1"
+                ;;
+            "-p"|"--partuuid")
+                shift
+                PARTUUID=1
+                ;;
+            *)
+                die "Unknown argument: $1. Please use --help for help."
+            ;;
+        esac
+        shift
+    done
+
+    rootfs_dir_host="$FC_BUILD_DIR"/rootfs
+    rootfs_dir_ctr="$CTR_FC_BUILD_DIR"/rootfs
+    flavour="bionic"
+    ROOTFS_NAME="$flavour.rootfs.ext4"
+    ROOTFS_PARTUUID_NAME="$flavour.rootfs_with_partuuid.ext4"
+    create_dir "$rootfs_dir_host"
+
+    # TODO: merge changes in vmm-reference
+    recipe_url="https://raw.githubusercontent.com/dianpopa/vmm-reference/rootfs_fc/resources/disk/make_rootfs.sh"
```

Comment:
Is this the correct url with dianpopa? Can we have a "generic" one?

Reply:
The needed changes are not yet merged so for the moment we have a link to my fork of rust-vmm/vmm-reference. When these changes will be merged https://github.com/rust-vmm/vmm-reference/pull/150, we can replace dianpopa with rust-vmm. What do you mean by "generic"?

Reply:
Thanks Diana! Is fine like that. By generic, I meant a repo that is org related, not user related like rust-vmm.

Reply:
Same suggestion as in the other PR where we should fix the commit ID.

Reply:
Looks like https://github.com/rust-vmm/vmm-reference/pull/150 was merged. Can you update the URL?